dpsim-models/include/dpsim-models/Attribute.h

```diff
@@ -361,9 +361,6 @@ namespace CPS {
 		 * @return true if the copy operation was successful, false otherwise
 		 */
 		virtual bool copyValue(AttributeBase::Ptr copyFrom) {
-			if (!this->isStatic()) {
```

Comment:
What problem occurred for dynamic attributes and how have you solved it?

Reply:
I think, I excluded dynamic attributes at first because they are usually meant to be computed from their dependencies and not set directly. However, since setting them directly is necessary when importing to a dynamic attribute with no dependencies configured (e. g. `mVoltageRef`), I removed the requirement for static attributes in 80f8f9eca423a1aadbc0a8fc24d9afa088a574e4. This works in general, but when importing to a dynamic attribute which actually has dependencies, the copied value will be overwritten immediately the next time the attribute is read, so the copy has no effect in this case.